src/main/java/frameworksdrivers/StudentDbGateway.java

```diff
@@ -67,24 +67,23 @@ public void getUser(LoginStudentDbRequestModel dbRequestModel) {
         }
     }
 
-    /**
-     * Method returns whether the username is unique meaning that it's not already present in the database.
-     *
-     * @param username is the username that is checked.
-     * @return boolean that indicates if username is unique.
-     */
-    @Override
-    public boolean isUnique(String username) {
-        return true;
-    }
-
```

Comment:
I agree that the isUnique method and the usernameExists method do the exact same thing.

So we should get rid of this method because you can just call the usernameExists but you'll have to check for the opposite: if the username exists then you can't create a new user. It removes redundant methods ! Good job !